engine/Gemfile

```diff
@@ -10,3 +10,7 @@ gem "sprockets-rails"
 
 # Start debugger with binding.b [https://github.com/ruby/debug]
 # gem "debug", ">= 1.0.0"
+
+group :development, :test do
+  gem "rspec-rails"
```

Comment:
Other repos (not all) had "rspec" explicitly added. It worked for me without so leaving it as it's necessary to add it.